requirements.txt

```diff
@@ -8,7 +8,6 @@ static3==0.5.1
 wsgiref==0.1.2
 -e git+https://github.com/funkybob/django-flatblocks.git#egg=django-flatblocks
 sendgrid-django
-psycopg2==2.5.4
```

Comment:
In other words, would it be a problem if I change the line 
```
-psycopg2==2.5.4
```
to 
```
-psycopg2
```
that removes the version requirement?